### PR TITLE
Closes #88: Add LCP Templates

### LINKED ISSUE #88: Add lcp templates
We need to add the following templates to the json of lcp => min-width not  handled 100% yet
- https://e2e.rocketlabsqa.ovh/lcp_picture_template/
desktop:
lcp{"/wp-content/rocket-test-data/images/test3.gif", '/rocket-test-data/images/lcp/testjpg2.jpg', '/rocket-test-data/images/testjpg3.jpg'}
atf{"/rocket-test-data/images/lcp/testwebp.webp", '/rocket-test-data/images/lcp/testjpeg.jpeg', '/rocket-test-data/images/paper.jpeg', "/test.png", '/rocket-test-data/images/1_BDCYb3Yx8exGZVu5lLRPNw.png', '/rocket-test-data/images/lcp/testavif.avif'}
enabled:false

- https://e2e.rocketlabsqa.ovh/lcp_picture_template2/
desktop:
lcp{'/rocket-test-data/images/paper.jpeg', '/rocket-test-data/images/lcp/testjpeg.jpeg', '/rocket-test-data/images/lcp/testwebp.webp'}
atf: {'/rocket-test-data/images/test3.gif', '/rocket-test-data/images/testjpg3.jpg', '/rocket-test-data/images/lcp/testjpg2.jpg'}
enabled:false
Mobile:
lcp{'/rocket-test-data/images/test3.gif', '/rocket-test-data/images/testjpg3.jpg', '/rocket-test-data/images/lcp/testjpg2.jpg'}

- https://e2e.rocketlabsqa.ovh/lcp_bg_img_in_section/
desktop/mobile:
lcp{https://e2e.rocketlabsqa.ovh/wp-content/uploads/2024/03/should_be_lcp_atf_image.jpg}
ATF []

- https://e2e.rocketlabsqa.ovh/lcp_section_template_relative/ 
desktop or mobile {lcp: '/wp-content/rocket-test-data/image/test3.webp', '/wp-content/rocket-test-data/image/file_example_JPG_100kB.jpg'}
atf:/wp-content/rocket-test-data/images/lcp/testjpg.jpg}

- https://e2e.rocketlabsqa.ovh/lcp_section_template2/
desktop or mobile {lcp: https://images.pexels.com/photos/67636/rose-blue-flower-rose-blooms-67636.jpeg?auto=compress&cs=tinysrgb&h=350
atf: /wp-content/rocket-test-data/images/lcp/testjpg.jpg}

### PR DESCRIPTION
**Pull request by Miraeld**
# Description

Fixes #88 

### Technical documentation

This PR adds the following templates for Desktop AND Mobile: 

- lcp_bg_img_in_section
- lcp_section_template_relative
- lcp_section_template2
- lcp_picture_template
- lcp_bg_relative_attribute_incss
- lcp_bg_img_in_header_template
- lcp_6647_svgbg_template
- lcp_6599_template2
- lcp_picture_media_type_mixed
- lcp_picture_relative

## Type of change
- [x] Enhancement (non-breaking change which improves an existing functionality).

# Checklists

## Feature validation

- [x] I validated all the Acceptance Criteria. If possible, provide sreenshots or videos.
- [x] I triggered all changed lines of code at least once without new errors/warnings/notices.

### CODE REVIEW

File: src/support/results/expectedResultsMobile.json [hunk 1]
```diff
@@ -228,5 +228,111 @@
...
+    "viewport": [
+      "/rocket-test-data/images/test3.gif",
+      "/rocket-test-data/images/testjpg3.jpg",
+      "/rocket-test-data/images/lcp/testjpg2.jpg"
+    ],
+    "enabled": true
+  },
+  "lcp_bg_img_in_section": {
+    "lcp": [
+      "https://e2e.rocketlabsqa.ovh/wp-content/uploads/2024/03/should_be_lcp_atf_image.jpg"
```
Comment (jeawhanlee) [inline src/support/results/expectedResultsMobile.json]:
```suggestion
      "/wp-content/uploads/2024/03/should_be_lcp_atf_image.jpg"
```

File: src/support/results/expectedResultsDesktop.json [hunk 2]
```diff
@@ -236,5 +236,110 @@
...
+    "viewport": [
+      "/rocket-test-data/images/test3.gif",
+      "/rocket-test-data/images/testjpg3.jpg",
+      "/rocket-test-data/images/lcp/testjpg2.jpg"
+    ],
+    "enabled": true
+  },
+  "lcp_bg_img_in_section": {
+    "lcp": [
+      "https://e2e.rocketlabsqa.ovh/wp-content/uploads/2024/03/should_be_lcp_atf_image.jpg"
```
Comment (jeawhanlee) [inline src/support/results/expectedResultsDesktop.json]:
```suggestion
      "/wp-content/uploads/2024/03/should_be_lcp_atf_image.jpg"
```

**Review by jeawhanlee — CHANGES_REQUESTED** [review]

**Comment by Miraeld** [comment]
Updated to comply latest feedback :)

**Review by jeawhanlee — DISMISSED** [review]

**Comment by jeawhanlee** [comment]
The viewport expectation for the template: `lcp_picture_template2` is wrong on mobile which causes a failure for that specific template. Manual testing shows this.

`Expected Viewport for mobile - /rocket-test-data/images/test3.gif for http://mike.e2e.rocketlabsqa.ovh/lcp_picture_template2 is not present in actual - [{"type":"picture","src":"http://mike.e2e.rocketlabsqa.ovh/wp-content/rocket-test-data/images/1_BDCYb3Yx8exGZVu5lLRPNw.png","sources":[{"srcset":"http://mike.e2e.rocketlabsqa.ovh/wp-content/rocket-test-data/images/lcp/testavif.avif","media":"","type":"","sizes":""}]},{"type":"picture","src":"http://mike.e2e.rocketlabsqa.ovh/test.png","sources":[]},{"type":"picture","src":"http://mike.e2e.rocketlabsqa.ovh/wp-content/rocket-test-data/images/lcp/testwebp.webp","sources":[{"srcset":"http://mike.e2e.rocketlabsqa.ovh/wp-content/rocket-test-data/images/paper.jpeg","media":"(max-width: 400px)","type":"","sizes":""},{"srcset":"http://mike.e2e.rocketlabsqa.ovh/wp-content/rocket-test-data/images/lcp/testjpeg.jpeg","media":"(max-width: 800px)","type":"","sizes":""}]}]`


`Expected Viewport for mobile - /rocket-test-data/images/testjpg3.jpg for http://mike.e2e.rocketlabsqa.ovh/lcp_picture_template2 is not present in actual - [{"type":"picture","src":"http://mike.e2e.rocketlabsqa.ovh/wp-content/rocket-test-data/images/1_BDCYb3Yx8exGZVu5lLRPNw.png","sources":[{"srcset":"http://mike.e2e.rocketlabsqa.ovh/wp-content/rocket-test-data/images/lcp/testavif.avif","media":"","type":"","sizes":""}]},{"type":"picture","src":"http://mike.e2e.rocketlabsqa.ovh/test.png","sources":[]},{"type":"picture","src":"http://mike.e2e.rocketlabsqa.ovh/wp-content/rocket-test-data/images/lcp/testwebp.webp","sources":[{"srcset":"http://mike.e2e.rocketlabsqa.ovh/wp-content/rocket-test-data/images/paper.jpeg","media":"(max-width: 400px)","type":"","sizes":""},{"srcset":"http://mike.e2e.rocketlabsqa.ovh/wp-content/rocket-test-data/images/lcp/testjpeg.jpeg","media":"(max-width: 800px)","type":"","sizes":""}]}]`


`Expected Viewport for mobile - /rocket-test-data/images/lcp/testjpg2.jpg for http://mike.e2e.rocketlabsqa.ovh/lcp_picture_template2 is not present in actual - [{"type":"picture","src":"http://mike.e2e.rocketlabsqa.ovh/wp-content/rocket-test-data/images/1_BDCYb3Yx8exGZVu5lLRPNw.png","sources":[{"srcset":"http://mike.e2e.rocketlabsqa.ovh/wp-content/rocket-test-data/images/lcp/testavif.avif","media":"","type":"","sizes":""}]},{"type":"picture","src":"http://mike.e2e.rocketlabsqa.ovh/test.png","sources":[]},{"type":"picture","src":"http://mike.e2e.rocketlabsqa.ovh/wp-content/rocket-test-data/images/lcp/testwebp.webp","sources":[{"srcset":"http://mike.e2e.rocketlabsqa.ovh/wp-content/rocket-test-data/images/paper.jpeg","media":"(max-width: 400px)","type":"","sizes":""},{"srcset":"http://mike.e2e.rocketlabsqa.ovh/wp-content/rocket-test-data/images/lcp/testjpeg.jpeg","media":"(max-width: 800px)","type":"","sizes":""}]}]`

**Review by jeawhanlee — APPROVED** [review]

**Comment by MathieuLamiot** [comment]
All good here on the new templates.

**Pull request merged**